all_test.go

```diff
@@ -720,6 +733,34 @@ func TestMaxSessionLimit(t *testing.T) {
 	}
 }
 
+func TestSqlSessionCache(t *testing.T) {
+	c := setup1(t)
+	defer Teardown(c)
+	c.MaxActiveSessions = 1
+
+	// Login first time
+	key, _, _ := c.Login(joeIdentity, joePwd)
```

Comment:
It is not. We login here to generate a session key and remove it from the memory. Then we test the session cache, which would then hit the db. I should probably change the name of the test to: "TestDBSessionCache"